examples/schelling_experimental/app.py

```diff
@@ -1,4 +1,4 @@
-from mesa.visualization.jupyter_viz import JupyterViz, Slider, make_text
+from mesa.visualization.solara_viz.py import Slider, SolaraViz, make_text
```

Comment:
This line doesn't work.

Reply:
Do you think because of the re-order? Ruff mandated that.

Reply:
Yes, removing .py fixed it.

Reply:
Fixed in 274a1bcdce13d86f992eed085093f8954400ad98